ui: removed metric function `setDefaultTime`

vladlos opened the pull request:
Removed setter the default graph time window based on the age of the older node in the cluster because we have default redux scale. That fixed our problem when we using the time dropdown on the metrics page and clicking on "10m" we get result with "6h" duration.

Resolves: #46145

Release justification: bug fixes and low-risk updates to new functionality

Release note (ui): default timescale on metrics page is always 10m. previously it defaulted to the age of the longest running node

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/46190)
<!-- Reviewable:end -->

dhartunian reviewed (approved):
Looks good. Didn't know we had this feature previously.
Going to just confirm with Piyush that we want that old feature removed.

Please add a Release Justification to commit and PR message so we can merge :)

> Reviewed 3 of 3 files at r1.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/46190#-:-M3Im-NE6ARLFVScR_n1:b-7hba0y)* status: :shipit: complete! 0 of 0 LGTMs obtained



<!-- Sent from Reviewable.io -->

dhartunian commented:
@elkmaster can you fix the merge conflict so we can merge?

dhartunian commented:
This will also need a release note: "default timescale on metrics page is always 10m. previously it defaulted to the age of the longest running node"

vladlos committed:
ui: removed metric function `setDefaultTime`

Removed setter the default graph time window based on the age of the older node in the cluster because we have default redux scale. That fixed our problem when we using the time dropdown on the metrics page and clicking on "10m" we get result with "6h" duration.

Resolves: #46145

Release justification: bug fixes and low-risk updates to new functionality

Release note (ui): default timescale on metrics page is always 10m. previously it defaulted to the age of the longest running node

dhartunian commented:
bors r+
